masonry_core/src/widgets/text_area.rs

```diff
@@ -101,6 +101,10 @@ pub struct TextArea<const USER_EDITABLE: bool> {
     /// Can be set using [`set_padding`](Self::set_padding).
     /// Immediate parent widgets should use [`with_padding_if_default`](Self::with_padding_if_default).
     padding: Padding,
+
+    // What key combination should trigger a newline insertion
+    // If this is set to `InsertNewline::OnEnter` then `Enter` will insert a newwline and _not_ trigger a `TextEntered` event
```

Comment:
Shouldn't this be a doc comment?

Also

> will insert a newwline and

has a typo in "newwline". Is this a `typos` failure @DJMcNab ?

Reply:
:facepalm: fixed

Reply:
Thanks for catching this. When we catch a type, we like to contribute it to https://github.com/crate-ci/typos - there's a monthly issue.
(It's always worth sanity checking with a search engine that there isn't a reason to use the typo version, before submitting)